packages/suite/src/reducers/wallet/cardanoStakingReducer.ts

```diff
@@ -25,7 +25,7 @@ export const initialState: State = {
         isFetchLoading: false,
         isFetchError: false,
     },
-    testnet: {
+    preview: {
```

Comment:
can you confirm that we do not need database migration because of these changes?

Reply:
Yes, it's ok.